.github/workflows/cpack.yml

```diff
@@ -56,131 +58,156 @@ jobs:
     env:
       ZITI_DEB_TEST_REPO: ${{ vars.ZITI_DEB_TEST_REPO || 'zitipax-openziti-deb-test' }}
       ZITI_RPM_TEST_REPO: ${{ vars.ZITI_RPM_TEST_REPO || 'zitipax-openziti-rpm-test' }}
-      VCPKG_BINARY_SOURCES: clear;files,${{ github.workspace }}/vcpkg_cache,readwrite
+      VCPKG_BINARY_SOURCES: clear;files,${{ github.workspace }}/vcpkg_cache/archives,readwrite
```

Comment:
moved the binary cache down a level to "archives" so that "downloads" and "installed" caches can live at the same level in the GH cache dir